examples/simple/handlers.go

```diff
@@ -47,3 +48,14 @@ func registerHTTPers(service service.Host) []uhttp.RouteHandler {
 		uhttp.NewRouteHandler("/", handler),
 	}
 }
+
+func simpleFilter() uhttp.FilterFunc {
```

Comment:
good point. I can implement uhttp.Filter with Apply(ctx, w, r, next) method. That might be better example.